hasura/graphql-engine
# Change default type placeholder text to type, fix 2394

### Description
Change the placeholder text for fields where we enter values (eg insert row) from 'text' to the type for types not handled specially.

### Affected components 

- Server
- CLI
- Docs
- Community Content
- Build System
- Tests
- Other (list it)

### Related Issues
https://github.com/hasura/graphql-engine/issues/2394

### Solution and Design
Changed the return from the constant string 'text' to the unadorned PostGres type
if the code doesn't explicitly handle the type.
Since this only affects "oddball" types, telling the user the type expected is, IMHO, the best option.

### Steps to test and verify
1. Start system normally
2. Create a table with an int autoincrement primary key and a second smallint column
3. Navigate to the insert row page, observe that the placeholder is now Int2, which is a better hint

### Limitations, known bugs & workarounds
Placeholders are a questionable UI feature, we should be deprecating them out of the interface.
See [this post by the prestigious UX design firm Nielsen Norman](https://www.nngroup.com/articles/form-design-placeholders/)
